spec/factories/plans.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: plans
+#
+#  id           :bigint           not null, primary key
+#  name         :string           not null
+#  target_limit :integer          not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+FactoryBot.define do
+  factory :plan do
+    sequence(:id) { |n| n + 2 }
```

Comment:
Hello, initially I didn't have anything, but I had to add it because otherwise it would generate an error in the tests. When I created a target with the factory, it told me that there was already one with that ID, so it occurred to me that instead of starting at 1, it should start at 2